prow/config.yaml

```diff
@@ -1208,7 +1208,7 @@ postsubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:0.2
+      - image: gcr.io/k8s-testimages/bazelbuild:0.12
         args:
         - "--branch=$(PULL_REFS)"
```

Comment:
with 0.12+ you need to remove this.

with @fejta's PR you need to have `--repo`.

0.13 now exists (with bazel 0.5.2).